doc/src/tutorials/physics/mechanics/rotating_disc_block_friction.rst

```diff
@@ -0,0 +1,160 @@
+.. _rotating_disc_block_friction:
+
+==========================
+A Block on a Rotating Disc
+==========================
+
+This page demonstrates how to use the functionalities in :obj:`sympy.physics.mechanics`
+and the **CoulombKineticFriction** actuator to model a sliding block on a rotating disc.
```

Comment:
Can you use a `:class:` reference?

Reply:
Sure I changed `**CoulombKineticFriction**` to :class:`~.CoulombKineticFriction`.